taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -29,22 +29,30 @@ const GoToProject = ({ clearNavBarLinks }: Props) => {
   const navigate = useNavigate();
 
   const { data, isPending, isError } = useFetch<ProjectsAPIResponse>(
-    `${API_URL}projects?status=OPEN`
+    `${API_URL}projects`
```

Comment:
Maybe, in another PR I think we will have to provide a way to filter by status !